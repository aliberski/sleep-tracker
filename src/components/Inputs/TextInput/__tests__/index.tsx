import 'react-native';
import React from 'react';

import { Input } from '../index';
import renderer from 'react-test-renderer';

describe('Button', () => {
  const mockFunction = jest.fn();
  const input = renderer.create(
    <Input
      value='123'
      onChangeText={mockFunction}
      label='test'
      error='error'
    />,
  );

  it('renders correctly', () => {
    const tree = input.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
