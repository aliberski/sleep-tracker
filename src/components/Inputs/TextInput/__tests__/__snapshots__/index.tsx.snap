// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "#000000",
        "fontFamily": "Roboto-Bold",
        "fontSize": 13,
        "paddingBottom": 5,
      }
    }
  >
    test
  </Text>
  <TextInput
    allowFontScaling={true}
    onChangeText={[MockFunction]}
    rejectResponderTermination={true}
    selectionColor="#6666EE"
    style={
      Object {
        "borderColor": "#B00020",
        "borderWidth": 1,
        "fontFamily": "Roboto-Regular",
        "fontSize": 15,
        "paddingHorizontal": 10,
        "paddingVertical": 15,
        "width": "100%",
      }
    }
    underlineColorAndroid="transparent"
    value="123"
  />
  <Text
    style={
      Object {
        "color": "#B00020",
        "fontFamily": "Roboto-Regular",
        "fontSize": 13,
        "paddingBottom": 5,
        "paddingTop": 5,
      }
    }
  >
    error
  </Text>
</View>
`;
